client/signup/jetpack-connect/new-site/index.jsx

```diff
@@ -0,0 +1,118 @@
+/**
```

Comment:
Heads up that this component name doesn't follow the guidelines of being the full namespace: should be `jetpack-new-site` like the classNames used below.